sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -36,6 +36,7 @@ import org.apache.spark.sql.catalyst.expressions.codegen.CodegenContext
 import org.apache.spark.sql.catalyst.expressions.objects.InitializeJavaBean
 import org.apache.spark.sql.catalyst.rules.RuleIdCollection
 import org.apache.spark.sql.catalyst.util.BadRecordException
+import org.apache.spark.sql.errors.DataTypeErrorsBase
```

Comment:
nit: We are already in the package `org.apache.spark.sql.errors` and this import is useless.